spec/examples/from_ical_spec.rb

```diff
@@ -103,6 +103,21 @@ module IceCube
       }.to raise_error(/Expecting number in \[-366, -1\] or \[1, 366\]/)
     end
 
+    it "should be able to return text correctly while using BYYEARDAY / BYSETPOS param" do
+      rule = ::IceCube::Rule.from_ical "FREQ=YEARLY;COUNT=12;BYYEARDAY=28,29,30;BYSETPOS=-1"
+      expect(rule.to_s).to eq("Yearly 12 times on the 28th, 29th, and 30th days of the year")
+    end
+
+    it "should be able to return text correctly while using BYMONTHDAY / BYSETPOS param" do
+      rule = ::IceCube::Rule.from_ical "FREQ=MONTHLY;COUNT=12;BYMONTHDAY=28,29,30;BYSETPOS=-1"
+      expect(rule.to_s).to eq("Monthly 12 times on the 28th, 29th, and 30th days of the month")
```

Comment:
Maybe we can patch these odd descriptions next 😁 

Since using `BYSETPOS` should make it `Monthly 12 times on the 28th, 29th, *or* 30th day of the month`, though theres probably a better way to explain it still.

Reply:
Good point! The clarity of the text is something we are actually addressing with our Ignition app based on the information of our schedule in the app and some simple regex 😄